packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -41,17 +53,26 @@ interface Props {
 
 const RetroTopic = (props: Props) => {
   const {isEmail, to, topic} = props
-  const {reflections, title, voteCount} = topic
+  const {reflections, title, voteCount, thread} = topic
   const imageSource = isEmail ? 'static' : 'local'
   const icon = imageSource === 'local' ? 'thumb_up_18.svg' : 'thumb_up_18@3x.png'
   const src = `${emailDir}${icon}`
   const grid = useEmailItemGrid(reflections, 3)
+  const commentLength = thread.edges.length
+  const commentLinkLabel =
+    commentLength === 0
+      ? 'No Comments'
+      : commentLength >= 101
+      ? 'See 100+ Comments'
+      : `See ${commentLength} ${plural(commentLength, 'Comment')}`
   return (
     <>
       <tr>
         <td align='center' style={{paddingTop: 20}}>
           <AnchorIfEmail href={to} isEmail={isEmail} style={topicThemeHeading}>
+            {'“'}
```

Comment:
IMO it was a little cleaner without quotes (i think we got rid of them in a previous revision?)